### PR TITLE
Updated overview page

### PR DESCRIPTION
**Pull request by ruthryi**
Fixed links and formatting

### CODE REVIEW

File: en/docs/GettingStarted/overview.md [hunk 1]
```diff
@@ -45,26 +45,26 @@ WSO2 API Manager provides a state-of-the-art web interface called **WSO2 API Pub
 
 The diagram above shows the common lifecycle activities of an API developer/manager.
 
--   #### Developer Portal
+#### Developer Portal
 
 ![]({{base_path}}/assets/attachments/103327648/126556772.png)
 
-The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to [self register](https://docs.wso2.com/display/AM260/Customizing+the+API+Store) , discover, evaluate, subscribe to and consume APIs.
+The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to [self register](../Learn/ConsumeAPI/Customizations/customizing-the-api-store.md) , discover, evaluate, subscribe to and consume APIs.
```
Comment (Mariangela) [inline en/docs/GettingStarted/overview.md]: Normally doc links are added as follows without .md
[Configuring API Manager Analytics](../../../../Learn/Analytics/configuring-apim-analytics/)
Pls check if this link works

File: en/docs/GettingStarted/overview.md [hunk 2]
```diff
@@ -45,26 +45,26 @@ WSO2 API Manager provides a state-of-the-art web interface called **WSO2 API Pub
...
+The API Gateway is a runtime, backend component (an [API proxy](https://docs.wso2.com/display/EI611/Working+with+APIs) ) developed using [WSO2 ESB](https://docs.wso2.com/display/EI611/WSO2+Enterprise+Integrator+Documentation). API Gateway secures, protects, manages, and scales API calls. It intercepts API requests, applies policies such as throttling and security using handlers, and manages API statistics. Upon validation of a policy, the Gateway passes web service calls to the actual backend. If the service call is a token request, the Gateway passes it directly to the [Key Manager](#key-manager).
```
Comment (Mariangela) [inline en/docs/GettingStarted/overview.md]: We need to link to the EI 6.5.0 docs as mentioned in [1].
[1] https://apim.docs.wso2.com/en/latest/GettingStarted/about-this-release/

File: en/docs/GettingStarted/overview.md (hunk 1 above)
Comment (ruthryi) [reply en/docs/GettingStarted/overview.md]: This works locally. And there are a couple of other pages with the same type of links

Comment (shaniR) [reply en/docs/GettingStarted/overview.md]: when you have .md, in the URL how does it show?

**Pull request merged**